browser/html/cool-help.html

```diff
@@ -144,6 +144,8 @@ <h2>Navigating in Spreadsheets</h2>
             <tr> <td class="function">Moves the cursor to the right edge of the current data range. If the column to the right of the cell that contains the cursor is empty, the cursor moves to the next column to the right that contains data.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Right Arrow</kbd></td> </tr>
             <tr> <td class="function">Moves the cursor to the top edge of the current data range. If the row above the cell that contains the cursor is empty, the cursor moves up to the next row that contains data.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Up Arrow</kbd></td> </tr>
             <tr> <td class="function">Moves the cursor to the bottom edge of the current data range. If the row below the cell that contains the cursor is empty, the cursor moves down to the next row that contains data.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Down Arrow</kbd></td> </tr>
+            <tr> <td class="function">Moves from current sheet to next sheet.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Alt</kbd><span class="kbd--plus">+</span><kbd>Page Up</kbd></td> </tr>
+            <tr> <td class="function">Moves from current sheet to previous sheet.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Alt</kbd><span class="kbd--plus">+</span><kbd>Page Down</kbd></td> </tr>
```

Comment:
Maybe better to have shorter wording and avoid using the verb move (to avoid misunderstandings about mocing the order of the sheets):

Go to next sheet.
Go to previous sheet.

Reply:
![image](https://github.com/CollaboraOnline/online/assets/61383886/2c4f4345-0e0c-4b86-acc7-3d4e3d0c15c9)